### DISCUSSION TITLE
Question about the final release version

### DISCUSSION THREAD

**Discussion by ZhaoxiongChen** [tabler/tabler#1984] [Q&A] [answered]
Hi, could you tell me when will the final release version come out?
By the way, I was wondering why the version number in the official website is 20 rather than 21, could you help me figure it out?

**Comment by dylan-tao** [comment, marked as answer]
> Hi, could you tell me when will the final release version come out? By the way, I was wondering why the version number in the official website is 20 rather than 21, could you help me figure it out?

@ZhaoxiongChen  I have sent the author an email with the updated version, but I haven't replied yet. Please reply after seeing it. I checked whether the submission record was specially deleted in the dist or 21 version. I don't know the author's intention.

@codecalm Please answer this question, thanks！Does the finished product package of the official website need to be updated to the latest version of 21?
